Modules/SubModule_Import/xASL_imp_TokenBackwardsCompatibility.m

```diff
@@ -0,0 +1,45 @@
+function imPar = xASL_imp_TokenBackwardsCompatibility(imPar)
```

Comment:
Not only token right, also the `dcm2nii_version` etc. is the `dcm2nii_version` adapted nowhere else? Seems like something that is very critical and shouldn't be "hidden" in this function?